src/SumoCoders/FrameworkCoreBundle/Tests/Composer/ScriptHandlerTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Tests\Composer;
+
+use SumoCoders\FrameworkCoreBundle\Composer\ScriptHandler;
+
+class ScriptHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected function getEvent()
+    {
+        $io = $this->getMockBuilder('\Composer\IO\ConsoleIO')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $io->method('isDecorated')
+            ->willReturn(true);
+
+        $this->assertTrue($io->isDecorated());
```

Comment:
Volgens mij test deze lijn niets (behalve dan testen of je mocking framework werkt...) Mag dus weg.

Van wat ik kan zien heeft die `runCommandOnlyInDevMode()` eigenlijk niet het hele Event object nodig, enkel de IO. Het opbouwen van een mock event met daarin een mock IO is eigenlijk al een weggever dat dit niet volledig klopt. Ik zou dus enkel IO injecten in die method.